include/raylib-nuklear.h

```diff
@@ -566,6 +566,12 @@ DrawNuklear(struct nk_context * ctx)
     nk_clear(ctx);
 }
 
+struct nk_raylib_input_keyboard_check {
+    int key;
+    int input_key;
+    bool other;
```

Comment:
I didn't pay too much attention to naming, but `other` could probably be more descriptive, such as `modifier` or something like that?